src/effects/builtin/whitenoiseeffect.h

```diff
@@ -0,0 +1,48 @@
+#pragma once
+
+#include <random>
+
+#include "effects/effectprocessor.h"
+#include "engine/effects/engineeffect.h"
+#include "engine/effects/engineeffectparameter.h"
+#include "engine/filters/enginefilterpansingle.h"
+#include "util/class.h"
+#include "util/defs.h"
+#include "util/sample.h"
+#include "util/samplebuffer.h"
+#include "util/types.h"
+
+class WhiteNoiseGroupState final : public EffectState {
+  public:
+    WhiteNoiseGroupState(const mixxx::EngineParameters& bufferParameters)
+      : EffectState(bufferParameters),
+        gen(rs()) {
+    }
+    ~WhiteNoiseGroupState() {
+    }
+
+    CSAMPLE_GAIN prev_drywet;
```

Comment:
```suggestion
    CSAMPLE_GAIN previous_drywet;
```
no need to abbreviate this